cumulusci/tests/util.py

```diff
@@ -131,22 +131,27 @@ class FakeSF:
     """Extremely simplistic mock of the Simple-Salesforce API
 
     Can be improved as needed over time.
-    In particular, __getattr__ is not implemented yet.
     """
 
     fakes = {}
 
     def describe(self):
-        return self._get_json("global_describe")
+        describe = self._get_json("global_describe")
+        core_sobjects = ["Account", "Contact", "Case"]
+        describe["sobjects"] = [sobject for sobject in describe["sobjects"] if sobject["name"] in core_sobjects]
+        return describe
+
+    def __getattr__(self, name: str):
```

Comment:
I thought this method would call _get_json(name). global_describe.json doesn't have fields in it. You need Contact.json, Account.json and Case.json.